노욱진/index.js

```diff
@@ -0,0 +1,36 @@
+const buttons = document.querySelectorAll("button");
+const displayFormula = document.querySelector(".js-cal-display-formula");
+const display = document.querySelector(".js-cal-display");
+
+const resultButton = document.querySelector(".js-result");
+const resetButton = document.querySelector(".js-reset");
+
+const INIT_NUM = "0";
+
+const resultNumber = () => {
+  return (display.innerHTML = eval(display.innerHTML));
+};
+
+const resetNumber = () => {
+  return (display.innerHTML = INIT_NUM), (displayFormula.innerHTML = INIT_NUM);
+};
```

Comment:
(궁금한 점 세번째에 대한 코드입니다)

Reply:
```suggestion
const resetNumber = () => {
  display.innerHTML = INIT_NUM;
  displayFormula.innerHTML = INIT_NUM;
};
```

좋은 코드의 기준은 여러가지가 있겠지만, 제가 생각하는 여러 기준들 중 하나는 `의미가 명확한` 코드인데요

단순히 코드의 양을 줄이기 위해서 이렇게 작성하는 것은 제 스타일이 아니긴해요 ㅋㅋㅋ!!

추가적으로 `return`문 또한 의미가 없다고 생각되어 걷어내는게 더 좋지 않을까 생각합니다

---

~~욱진씨 정도의 실력이라면~~ 코드의 수를 적게 써야하는 경우가 아니라면,

좋은 코드는 무엇인지 고민해 보고 적용해 보는게 좋을 거 같아요

일반적으로 좋은 코드는 `읽기 좋은 코드`라고 들어보셨을 텐데 왜 읽기 좋은 코드가 좋은 코드인지도 고민해 보시면

조금 더 성장하시지 않을까 생각해유 👍

Reply:
코드의 수가 적다고 좋은 코드가 아니였군용..
항상 피드백 해주셔서 감사할 뿐입니다요.. 🙇‍♂️